internal/clients/http/azure/azure_client.go

```diff
@@ -3,17 +3,23 @@ package azure
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/authorization/armauthorization"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armsubscriptions"
 	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/clients/http"
 	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/ptr"
 	"github.com/RHEnVision/provisioning-backend/internal/telemetry"
 )
 
+const AZURE_SERVICE_ACCOUNT_OBJECT_ID = "bf9c3e0e-4bbd-425b-adb1-f005d3dfe9f9"
```

Comment:
Ugly, unfortunatelly this is the attribute we do not have in config, we do have application ID.

We could:
a) resolve applicationID to objectID through azure sdk,
b) store the applicationID in config as new value used just for this check.

Reply:
Storing it in the config makes sense, assuming it does not change too often.

Reply:
Oki, moved to config